packages/commonwealth/server/migrations/20240729220841-set-is-welcome-onboarding.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.query(`
+      UPDATE "Users"
+      SET is_welcome_onboard_flow_complete = true
+      WHERE profile ->> 'name' != 'Anonymous' AND profile ->> 'name' IS NOT NULL;
```

Comment:
can we use `DEFAULT_NAME` from "@hicommonwealth/shared"

Reply:
I'm not sure that I can import from libs in migrations. Additionally, migrations are meant to be correct at a specific point in time. If I use `DEFAULT_NAME` the effect of the migration may be different in the future if the variable is updated.

Reply:
makes sense, thanks.